cli/lib/platforms/cordova/index.js

```diff
@@ -0,0 +1,109 @@
+'use strict';
+
+const util = require('./util');
+const cfg = require('../../cfg');
+const log = require('../../log');
+const inquirer = require('inquirer');
+const mkdirSync = require('mkdirp').sync;
+const fs = require('fs');
+const join = require('path').join;
+const et = require('elementtree');
+
+/**
+ * Locations that a user can specify to use for a cordova binary
+ */
+const LOCATIONS = exports.LOCATIONS = ['local', 'global', 'custom'];
+
+/**
+ * Initialise the current Cordova project with Red Hat Mobile Hot Code Push
+ */
+exports.initialise = function () {
+  return util.installHcpPlugin()
+    .then(addHooks)
+    .then(updateConfigXml);
+};
+
+/**
+ * Asks questions required to complete Cordova setup
+ * @return {Promise<Object>}
+ */
+exports.askQuestions = function () {
+  log.debug('asking extra setup questions if required');
+  return inquirer.prompt([
+      {
+        type: 'list',
+        name: 'cordova.location',
+        message: 'Where is the Cordova executable located? Local is recommended to avoid versioning issues.',
```

Comment:
This allows user to choose cordova installed in node_modules, global, or custom binary location.